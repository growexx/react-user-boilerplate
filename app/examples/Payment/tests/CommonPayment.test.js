import React from 'react';
import { render } from 'react-testing-library';

import CommonPaymentInput from '../CommonPaymentInput';

describe('<CommonPaymentInput />', () => {
  it('should render with continue btn disable', () => {
    const props = {
      inputPlaceholder: 'abc',
      onInputChange: () => {},
      onLinkClick: () => {},
      onContinueClick: () => {},
      continueBtnDisable: false,
      inputValue: 'abc',
    };
    const {
      container: { firstChild },
    } = render(<CommonPaymentInput {...props} />);
    expect(firstChild).toMatchSnapshot();
  });
});
