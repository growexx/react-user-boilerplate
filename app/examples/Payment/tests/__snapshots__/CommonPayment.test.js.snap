// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommonPaymentInput /> should render with continue btn disable 1`] = `
<div
  class="ant-card common-payment-main"
>
  <div
    class="ant-card-body"
  >
    <div
      class="payment-main-card"
    >
      <span
        class="ant-input-affix-wrapper"
      >
        <input
          class="ant-input"
          placeholder="abc"
          type="text"
          value="abc"
        />
        <span
          class="ant-input-suffix"
        >
          <button
            class="ant-btn ant-btn-link"
            type="button"
          >
            <span>
              Link
            </span>
          </button>
        </span>
      </span>
      <button
        class="ant-btn ant-btn-primary continue-btn"
        type="button"
      >
        <span>
          Continue
        </span>
      </button>
    </div>
  </div>
</div>
`;
